seedDB/Update-Seed-Database.md

```diff
@@ -0,0 +1,22 @@
+# Update the seed database
+To update the seed database to a recent version you should follow these steps:
+
+1- Set up a new instance of the cBioPortal database, by using the most recent available seed database: [more information](https://github.com/cbioportal/datahub/blob/master/seedDB/README.md)
+
+2- Run the migration script: [more information](https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-your-cBioPortal-installation.md#running-the-migration-script)
+
+3- Move to the folder where you want to save the seed files and generate them. To do that, use the following commands (assuming that the database is running on port 8306):
+
+```shell
+mysqldump -u cbio -pP@ssword1 -P 8306 --host 127.0.0.1 --ignore-table cbioportal.pdb_uniprot_alignment --ignore-table cbioportal.pdb_uniprot_residue_mapping cbioportal > seed-cbioportal_no-pdb_hg19.sql
```

Comment:
Please add `--no-create-info`. We don't want to include the schema in this file. When importing, we get the schema from cgds.sql